src/workbench/material.ts

```diff
@@ -362,6 +362,9 @@ export function materialWorkbench(
     "editor.foldBackground": `${palette.bg5}80`,
     "editorBracketMatch.border": `${palette.bg}00`,
     "editorBracketMatch.background": `${palette.grey0}80`,
+    "editorBracketHighlight.foreground1": `${palette.yellow}`,
+    "editorBracketHighlight.foreground2": `${palette.red}`,
+    "editorBracketHighlight.foreground3": `${palette.blue}`,
```

Comment:
There are 6 available foreground tokens but you only added 3. Here are some suggestions:

```typescript
    "editorBracketHighlight.foreground1": `${palette.red}`,
    "editorBracketHighlight.foreground2": `${palette.yellow}`,
    "editorBracketHighlight.foreground3": `${palette.green}`,
    "editorBracketHighlight.foreground4": `${palette.blue}`,
    "editorBracketHighlight.foreground5": `${palette.orange}`,
    "editorBracketHighlight.foreground6": `${palette.purple}`,
    "editorBracketHighlight.unexpectedBracket.foreground": `${palette.grey1}`,
```

The key here is that you need to make the distinction between two adjacent parentheses large enough.